update_actions.sh

```diff
@@ -0,0 +1,44 @@
+
+#!/bin/bash
+set -e
+
+VERSION=$1
+
+if [ -z "$VERSION" ]; then
+        echo "missing version"
+        exit 1
+fi
+
+actionsRepos=(okteto/delete-namespace
+              okteto/build
+              okteto/destroy-preview
+              okteto/deploy-preview
+              okteto/deploy-stack
+              okteto/namespace
+              okteto/pipeline
+              okteto/push
+              okteto/create-namespace
+              okteto/destroy-pipeline
+              okteto/login
+              okteto/destroy-stack
+              okteto/apply)
+              
+for repo in "${actionsRepos[@]}"
+do
+    echo "$repo"
+    git clone --depth 1 https://github.com/$repo.git
+    repoPath=${repo#"okteto/"}
```

Comment:
since all of them will be in okteto, maybe it's easier for the list to only be the names of the repos, and in the clone URL add okteto?